lms/djangoapps/learner_home/waffle.py

```diff
@@ -22,7 +23,25 @@
 )
 
 
-def should_redirect_to_learner_home_mfe():
-    return configuration_helpers.get_value(
+def should_redirect_to_learner_home_mfe(user):
+    """
+    Redirect a percentage of learners to Learner Home for experimentation.
+
+    Percentage is based on the LEARNER_HOME_MFE_REDIRECT_PERCENTAGE setting.
+    """
+
+    is_learning_mfe_enabled = configuration_helpers.get_value(
         "ENABLE_LEARNER_HOME_MFE", ENABLE_LEARNER_HOME_MFE.is_enabled()
     )
+
+    learning_mfe_redirect_percent = configuration_helpers.get_value(
+        "LEARNER_HOME_MFE_REDIRECT_PERCENTAGE",
+        settings.LEARNER_HOME_MFE_REDIRECT_PERCENTAGE,
+    )
+
+    # Redirect when 1) Learner Home MFE is enabled and 2) a user falls into the
+    # target range for experimental rollout.
+    if is_learning_mfe_enabled and user.id % 100 < learning_mfe_redirect_percent:
```

Comment:
Will this run the risk of uneven and random user selection?

Reply:
Has already been vetted by data as being a fair, unbiased selection.